BookingsAPI/Bookings.DAL/Commands/HearingService.cs

```diff
@@ -37,8 +47,9 @@ public HearingService(BookingsDbContext context)
             _context = context;
         }
         
-        public async Task AddParticipantToService(VideoHearing hearing, List<NewParticipant> participants)
+        public async Task<List<Participant>> AddParticipantToService(VideoHearing hearing, List<NewParticipant> participants)
```

Comment:
Do we need to return a list of participants when you have the usernames in your request model?